src/webui/www/private/scripts/misc.js

```diff
@@ -28,6 +28,108 @@
 
 "use strict";
 
+/**
+ * @type Record<string, {locale: string, options: {}}>
+ */
+const DateFormatOptions = Object.freeze({
+    "MM/dd/yyyy, h:mm:ss AM/PM": {
+        locale: "en-US",
+        options: {
+            year: "numeric",
+            month: "2-digit",
+            day: "2-digit",
+            hour: "numeric",
+            minute: "2-digit",
+            second: "2-digit",
+            hour12: true
+        }
+    },
+    "MM/dd/yyyy, HH:mm:ss": {
+        locale: "en-US",
+        options: {
+            year: "numeric",
+            month: "2-digit",
+            day: "2-digit",
+            hour: "2-digit",
+            minute: "2-digit",
+            second: "2-digit",
+            hour12: false
+        }
+    },
+    "dd/MM/yyyy, HH:mm:ss": {
+        locale: "en-GB",
+        options: {
+            year: "numeric",
+            month: "2-digit",
+            day: "2-digit",
+            hour: "2-digit",
+            minute: "2-digit",
+            second: "2-digit",
+            hour12: false
+        }
+    },
+    "yyyy-MM-dd HH:mm:ss": {
+        locale: "sv-SE",
```

Comment:
In fact this is also the recommended format for Canada. But some people/projects still haven't updated the Canadian locale `en-CA`.